dashboard/utils_test.py

```diff
@@ -645,6 +653,43 @@ def test_count_courses_passed_fa(self):
         )
         assert mmtrack.count_courses_passed() == 1
 
+    @override_settings(FEATURES={"FINAL_GRADE_ALGORITHM": "v1"})
+    def test_count_courses_passed_normal(self):
```

Comment:
Is it worth suffixing this test with `_v1` just like the others or is your thought that we'll eventually sunset the `_v0` code and you don't want to have to rename everything down the road?